ipam/allocator.go

```diff
@@ -70,6 +70,8 @@ func NewAllocator(lcDs, glDs datastore.DataStore) (*Allocator, error) {
 		}
 	}
 
+	a.checkConsistency(localAddressSpace)
```

Comment:
I thought of a flag and a variable to be stored in the store to act as a lock, but still it would not cover the case where user upgrade one host at the time while older version daemona are running on other host.